### DISCUSSION TITLE
Less verbose output between versions

### DISCUSSION THREAD

**Discussion by MGatner** [composer-unused/composer-unused#228] [Q&A]
I was using Phive to install `composer-used` during GitHub Actions CI:
```
sudo phive --no-progress install --global --trust-gpg-keys F4D32E2C9343B2AE composer-unused 
composer-unused -vvv --profile --ansi --no-interaction --no-progress --excludePackage=php
```
Which resulted in version `2.1.3` and the expected "profile" and "verbose" output (https://github.com/tattersoftware/codeigniter4-workflows/runs/3998061872?check_suite_focus=true):
```
Box Requirements Checker
========================

> Using PHP 8.0.11
> PHP is using the following php.ini file:
  /etc/php/8.0/cli/php.ini

> Checking Box requirements:
  ✔ The application requires the version ">=7.3 || ^8.0" or greater.
  ✔ The package "composer/ca-bundle" requires the extension "openssl".
  ✔ The package "composer/ca-bundle" requires the extension "pcre".
  ✔ The package "nikic/php-parser" requires the extension "tokenizer".
  
 [OK] Your system is ready to run the application.                              

Running 2.1.3 (2021-06-09 16:31:20) with PHP 8.0.11 on Linux / 5.8.0-1042-azure
```

I noticed that `shivammathur/setup-php` has `composer-unused` available as a tool (very cool, by the way) so switched to that, which resulted in the newer version `2.1.9` but far less verbose output with the same switches (https://github.com/tattersoftware/codeigniter4-workflows/runs/3998136726?check_suite_focus=true):
```
Running 2.1.9 (2021-10-05 09:47:38) with PHP 8.0.11 on Linux / 5.8.0-1042-azure
```

**Comment by icanhazstring** [comment]
Hi, maybe I need to check how `shivammathur/setup-php` is building the `phar` file.
Also the phar packaging and cli options seem to interfere with one another. Which is not what I intended to happen :)

Let me have a look at this 👍

**Reply by MGatner** [reply]
Thanks mate! Appreciate all the rapid responses. Can I buy you a drink? PayPal.me accurate?

**Reply by icanhazstring** [reply]
Happy to help :)
And sure: https://paypal.me/icanhazstring
Much appreciated 👍
